operations/rack_resiliency/hpe-critical-services.md

```diff
@@ -0,0 +1,61 @@
+
+# HPE Critical Services
+
+| Name                                               | Resource Type            | Namespace                |
```

Comment:
Do we really want to just list all of the default critical services in here? Things like this are generally a bad idea because they are extremely prone to being out of date as people make adjustments to the service, and forget this page exists.